app/src/main/kotlin/cz/covid19cz/erouska/utils/DeviceInfo.kt

```diff
@@ -1,19 +1,25 @@
 package cz.covid19cz.erouska.utils
 
+import android.annotation.SuppressLint
+import android.content.Context
 import android.os.Build
 import com.jaredrummler.android.device.DeviceName
-import cz.covid19cz.erouska.App
+import org.koin.core.KoinComponent
+import org.koin.core.get
 import java.util.*
 
-object DeviceInfo {
+object DeviceInfo : KoinComponent {
```

Comment:
From my perspective, I'd rather turn all of this helper/manager classes into the classes instead of objects, and inject all of the dependencies properly via a constructor. If you're ok with that change, I'm ready to do it. I tried to avoid some significant changes in the current codebase.

In this specific case, I like your first suggestion, but objects don't have constructors, so I can't use constructor injection. A second suggestion is simple, but since `DeviceInfo` is used in `ViewModel`, I would prefer not to pass `Context` as an argument and leave VM without Android dependencies.

Reply:
Let's just convert both objects into classes with constructor injection?

Reply:
@simekadam updated.